packages/docusaurus-theme-classic/codeTranslations/vi.json

```diff
@@ -0,0 +1,55 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "Đóng",
+  "theme.CodeBlock.copied": "Đã sao chép",
+  "theme.CodeBlock.copy": "Sao chép",
+  "theme.CodeBlock.copyButtonAriaLabel": "Sao chép code vào bộ nhớ tạm (clipboard)",
```

Comment:
> clipboard

@quanengineering it is strictly necessary to specify the original word in parentheses or we can avoid it for brevity?

Reply:
At first, I think some users may find it difficult to understand some technical terms so I keep original word in parentheses in some place. For now, I think most users will understand, so I will remove original words.

Reply:
@lex111 I have just removed all original word.